node/common/client_config.ts

```diff
@@ -281,7 +281,7 @@ export class ClientConfig {
     ] as number;
   }
 
-  getAuditorLinearizableValidationContractId(): string | undefined | null {
+  getAuditorLinearizableValidationContractId(): string {
```

Comment:
we will have a default value `validate-ledger` anyway, so this part can be simplified.